mandatory/2-function-creation.js

```diff
@@ -6,7 +6,12 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+    if (num === "number" && num % 2 === 0 && num <= 100) {
```

Comment:
We don't want to check if num is equal to "number". We want to check if num's **type** is equal to "number"